airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -126,7 +126,7 @@ def get_connection_form_widgets(cls) -> dict[str, Any]:
         from wtforms import BooleanField, StringField
 
         return {
-            "use_beeline": BooleanField(lazy_gettext("Use Beeline"), default=False),
+            "use_beeline": BooleanField(lazy_gettext("Use Beeline"), default=True),
```

Comment:
This requires major release as it consider a breaking change isn't it?

Reply:
Why is it viewed as a breaking change? Would change in default value be that major?

Reply:
Got the information from slack